<launch>
	<node pkg="Project1" type="reset_Vrep.py" name="Reset_Vrep"/>

	<node pkg="Project1" type="Project1_node" name="Project1" output="screen">
		launch-prefix="xterm -e gdb --args"
		<param name="base_frame" value="/world" />
		<param name="max_range" value="4.5" />
		<param name="stepX" value="0.05"/>
		<param name="stepY" value="0.05"/>
		<param name="W_OBST_MAX2" value="0.5"/>
		<param name="distance_secu" value="0.2"/>
		<remap from="/Project1/scans" to="/vrep/depthSensor"/>
	</node>
</launch>
